README.md

```diff
@@ -44,10 +44,12 @@ Here are some commands you can run:
 | yarn flow:check             | Explicitly check for Flow errors and exit                                                                  |
 | yarn flow:dev               | Continuously check for Flow errors                                                                         |
 | yarn eslint                 | Lint the JS                                                                                                |
+| yarn snyk                   | Run [snyk][] (without a command)                                                                           |
```

Comment:
You could also just link these to the Synk section with [anchors](https://gist.github.com/asabaylus/3071099).